plugins/graphql-kotlin-maven-plugin/build.gradle.kts

```diff
@@ -86,6 +86,7 @@ tasks {
         mustRunAfter("startWireMock")
     }
     val integrationTest by register("integrationTest") {
+        dependsOn(publishToMavenLocal)
```

Comment:
should this be `dependsOn("publishToMavenLocal")`? i.e. `dependsOn` takes as an input vararg paths, 

`publishToMavenLocal` is a task so wondering whether its toString resolves as its path....

Reply:
This change only adds the dependency on publishing maven-plugin artifact. I believe we probably have to specify all dependencies explicitly....

```kotlin
dependsOn(":graphql-kotlin-types:publishToMavenLocal")
dependsOn(":graphql-kotlin-client:publishToMavenLocal")
dependsOn(":graphql-kotlin-ktor-client:publishToMavenLocal")
dependsOn(":graphql-kotlin-spring-client:publishToMavenLocal")
dependsOn(":graphql-kotlin-plugin-core:publishToMavenLocal")
dependsOn("publishToMavenLocal")
```